backend/src/threads/repository/review_repository.py

```diff
@@ -0,0 +1,106 @@
+import logging
+import uuid
+from typing import List, Optional
+
+from sqlalchemy.exc import SQLAlchemyError
+from sqlalchemy.orm import Session, joinedload
+
+from src.threads.models import Review
+from src.threads.schemas.model_schema import (
+    ReviewSchema,
+    ReviewUpdate,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class ReviewRepository:
+    def create_review(self, review: ReviewSchema, db: Session) -> bool:
+        try:
+            with db.begin_nested():
+                new_review = Review(
+                    user_id=review.user_id,
+                    attraction_id=review.attraction_id,
+                    rating=review.rating,
+                    price=review.price,
+                    time_spent=review.time_spent,
+                    title=review.title,
+                    description=review.description,
+                )
+                db.add(new_review)
+            db.commit()
+            return True
+        except SQLAlchemyError as error:
+            db.rollback()
+            logger.error("Error create review: %s", error)
```

Comment:
Do we need apply begin_nested() method ? I think in creation there is no pending transactions so maybe we could use with db.begin()